packages/core/src/api/profile.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {AuthClient} from 'src/auth-client';
+import AsgardeoUIException from 'src/exception';
+import {MeAPIResponse} from 'src/models/me-api-response';
+
+const getProfileInformation = async (): Promise<MeAPIResponse> => {
```

Comment:
Let's add doc comments to this function.

Reply:
added with b7bfa979ac040673bcece61fb66bf3fa76616953